Tasks/AzureRmWebAppDeployment/AzureRMUtil.ts

```diff
@@ -140,7 +148,18 @@ function getUpdateHistoryRequest(webAppPublishKuduUrl: string, isDeploymentSucce
         throw new Error(tl.loc('CannotupdatedeploymentstatusuniquedeploymentIdCannotBeRetrieved'));
     }
 
-    var message = "Updating Deployment History For Deployment " + buildOrReleaseUrl;
+    var message = JSON.stringify({
+		type : type,
+		commitId : commitId,
+		buildId : buildId,
```

Comment:
Do you also have information about Build Number and Release Number. If yes, add them as well